docs/swagger.yaml

```
@@ -494,6 +501,53 @@ paths:
+            type: string
+        "400":
+          description: Bad Request
+        "500":
+          description: Internal Server Error
+      summary: Creates a new temporary data entry
+      tags:
+      - tempdata
+    put:
+      description: Retrieve temporary data by UUID and delete it
+      parameters:
+      - description: UUID of the temporary data
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)